src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToGrid/SetSelectedTarget.java

```diff
@@ -0,0 +1,34 @@
+package edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+public abstract class SetSelectedTarget extends InstantCommand {
```

Comment:
Raz - This comment is for the entire class:
Why do we save as a member the side multiplier?
Also, why does the class need to receive the alliance? It can get itself, at any time.
I think that there is a need for a function that will get the "side multiplier" that is taking in considration both alliance and the "side multiplier", and the result of that function is the thing that would make sence to keep as a member of the function.
This function can be called in the constructor, or in the initalize and than you can keep as member the moveRight (by the way, I don't love that we pass a boolean here, but I can live with it here).

It might wasn't that clear, if you have questions, text me.

Reply:
Now after I read the SetSelectedTarget<Left/Right>, I do have an issue with this class.
If the class is setting the selected target, it should get the location to set to. It's a diffrent thing to decide where to set to. So either change the class name to something more clear, or it should maybe only recive the location to set to, and the logic to decide where to set it should be in a diffrent class (I prefer the second).